bolt/organization.go

```diff
@@ -234,7 +238,19 @@ func (c *Client) FindOrganizations(ctx context.Context, filter influxdb.Organiza
 		}
 	}
 
-	return os, len(os), nil
+	// Sorting
+	influxdb.SortOrganizations(opts, orgs)
```

Comment:
Just a comment.
Should the organizations which are sorted contain `AllOrganizations[:offset]` ?